src/types/protocols-types.ts

```diff
@@ -125,6 +125,11 @@ export type ProtocolRuleSet = {
    * by setting `protocolRole` property to the protocol path of the $contextRole record.
    */
   $contextRole?: boolean;
+  // If $size is set, the record size in bytes must be within the limits.
```

Comment:
Suggestion: modify comment style in this case so that `jsdoc` will pick up the comment in the IDE.
```suggestion
  /**
   * If $size is set, the record size in bytes must be within the limits.
   */
```